packages/assets-controllers/src/NftController.ts

```diff
@@ -1155,18 +1171,26 @@ export class NftController extends BaseController<NftConfig, NftState> {
    * @param asset.tokenId - The ID of the asset.
    * @param type - The asset type.
    * @param origin - Domain origin to register the asset from.
-   * @param networkClientId - The networkClientId that can be used to identify the network client to use for this request.
+   * @param options - Options bag.
+   * @param options.networkClientId - The networkClientId that can be used to identify the network client to use for this request.
+   * @param options.userAddress - The address of the account where the NFT is being added.
    * @returns Object containing a Promise resolving to the suggestedAsset address if accepted.
    */
   async watchNft(
     asset: NftAsset,
     type: NFTStandardType,
     origin: string,
-    networkClientId?: NetworkClientId,
+    {
+      networkClientId,
+      userAddress = this.config.selectedAddress,
+    }: {
+      networkClientId?: NetworkClientId;
+      userAddress?: string;
+    } = {
+      userAddress: this.config.selectedAddress,
+    },
```

Comment:
```suggestion
    } = {},
```


I don't believe the default empty object value and default object property values are both needed

Reply:
I think they are because they're two different cases. 1) If the method is not passed an options object at all, and 2) if the method is passed an options object but it does not contain a userAddress.